pkg/controllers/tagging/tagging_controller.go

```diff
@@ -269,6 +274,7 @@ func (tc *Controller) process() bool {
 		}
 
 		tc.workqueue.Forget(obj)
+		nodeTaggingDelay.Observe(time.Since(currentNode.CreationTimestamp.Time).Seconds())
```

Comment:
I think we want this within tagEc2Resource, around: https://github.com/kubernetes/cloud-provider-aws/blob/9358055c45bfad50f7d972571c1efbe5fa447f9f/pkg/controllers/tagging/tagging_controller.go#L343

Since we only want to record it when we're tagging (not un-tagging) and we don't want to record it unless tags are actually applied